common/data/sentencePlanningApi.js

```diff
@@ -29,6 +29,21 @@ const setSentencePlanComment = async (sentencePlanId, comment, token) => {
   return putData(path, token, comment)
 }
 
+const getSentencePlanObjective = async (sentencePlanId, objectiveId, token) => {
+  const path = `${url}/sentenceplans/${sentencePlanId}/objectives/${objectiveId}`
+  return getData(path, token)
+}
+
+const setNewSentencePlanObjective = async (sentencePlanId, objective, token) => {
```

Comment:
Semantics: `set` sounds like a singular, could we just call it `addSentencePlanObjective`?